models/project.ini

```diff
@@ -23,7 +23,7 @@ width = 1/4
 
 [fields.maturity]
 label = Maturity
-type = select
+type = checkboxes
```

Comment:
From a data modelling perspective, I'm wondering if it might be better to have a completely independent boolean for attic status. Allowing a data model where something can be both "alpha" and "beta", just so we can have "attic" and "production" seems... wrong.